SWLOR.Game.Server/Service/PlayerStatService.cs

```diff
@@ -497,6 +497,12 @@ public static EffectiveItemStats GetPlayerItemEffectiveStats(NWPlayer player, NW
 
                     int itemAC = item.CustomAC;
                     itemAC = CalculateAdjustedValue(itemAC, item.RecommendedLevel, skillRankToUse, 0);
+
+                    if (itemAC > maxAC)
+                    {
+                        //itemAC = maxAC;
+                        item.SetLocalInt("CUSTOM_ITEM_PROPERTY_AC",itemAC = maxAC);
```

Comment:
Pull from the item.CustomAC property instead of getting the raw int by variable name.

Reply:
Not exactly sure what you mean here...
The reason I am using setlocalint is so the item is set to the right AC on the examine window  just to avoid confusion to players. Otherwise it can say you get 40 ac but you only get 12(or 14).

Reply:
There's wrapper code around the CustomAC property which handles conversion of item properties to local ints and some other stuff. It's dangerous to access the variables directly as that conversion code doesn't take place. So instead of doing a SetLocalInt, do item.CustomAC = maxAC;